src/components/MdxComponents/Contributors.tsx

```diff
@@ -37,7 +47,12 @@ const Contributors = () => {
         return (
           // eslint-disable-next-line react/jsx-no-target-blank
           <a key={contributor.id} href={contributor.html_url} target='_blank'>
-            <img alt={contributor.login} src={contributor.avatar_url} />
+            <img
+              alt={contributor.login}
+              src={contributor.avatar_url}
+              width="100px"
```

Comment:
Why do we need image's width, height attribute? I think we can set it from CSS component.
Could you please check it?

Reply:
It can make it both. But I will follow your guide by convert to CSS component. 😁